metadata/metadata.go

```diff
@@ -1622,6 +1705,23 @@ func (serv *MetadataServer) ListTrainingSets(_ *pb.Empty, stream pb.Metadata_Lis
 
 func (serv *MetadataServer) CreateTrainingSetVariant(ctx context.Context, variant *pb.TrainingSetVariant) (*pb.Empty, error) {
 	variant.Created = tspb.New(time.Now())
+	triggers := variant.Trigger
+	for _, t := range triggers {
+		trigger, err := serv.lookup.Lookup(ResourceID{Name: t, Type: TRIGGER})
+		if err != nil {
+			return nil, fmt.Errorf("trigger does not exist: %v", err)
+		}
+		asserted_trigger, ok := trigger.(*triggerResource)
```

Comment:
what is an asserted trigger?

Reply:
Once it is verified that trigger is of type Trigger Resource, its named asserted trigger